src/main/java/gift/controller/category/CategoryController.java

```diff
@@ -50,7 +51,7 @@ public ResponseEntity<String> updateCategory(@PathVariable(name = "id") Long cat
     @DeleteMapping("/{id}")
     public ResponseEntity<String> deleteCategory(@PathVariable(name = "id") Long categoryId) {
         categoryService.deleteCategory(categoryId);
-        return ResponseEntity.ok(categoryId + "번 카테고리가 삭제되었습니다!");
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).body(categoryId + "번 카테고리가 삭제되었습니다!");
```

Comment:
요렇게도 활용할 수 있으니 참고해주세요ㅎㅎ
```suggestion
        return ResponseEntity.noContent().body(categoryId + "번 카테고리가 삭제되었습니다!");
```

Reply:
noContent도 메서드로 존재했네요. 감사합니다.